app/controllers/sessions_controller.rb

```diff
@@ -51,6 +51,10 @@ def omniauth
       @auth = request.env['omniauth.auth']
       @user_exists = check_user_exists
 
+      if !@user_exists && @auth['provider'] == "twitter"
+        return redirect_to root_path, flash: { alert: I18n.t("registration.deprecated.twitter_signup") }
+      end
```

Comment:
This can be done in 1 line (see the line below it)

Reply:
Then it goes over the line column limit

Reply:
Hm. Good point. In that case I would leave it as is